docs/sql/tutorials/monthly-expediture-timegpt.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Forecasting Monthly Expenditures with TimeGPT
+sidebarTitle: House Sales with TimeGPT
+---
+
+## Introduction
+
+In this tutorial, we introduce Nixtla’s TimeGPT integration which offers the first foundational model for time series forecasting. Follow along to see how it works.
+
+## Prerequisites
+
+### MindsDB Setup
+
+One way is to sign up for an account at [MindsDB Cloud](https://cloud.mindsdb.com/). It is a convenient option as it doesn’t require any installation procedures. You can find the details [here](/setup/cloud).
+
+Alternatively, visit our docs and follow the instructions to manually set up a local instance of MindsDB via [Docker](/setup/self-hosted/docker) or [pip](/setup/self-hosted/pip/source). You can also set up MindsDB on AWS following [this instruction set](https://aws.amazon.com/marketplace/pp/prodview-2nmhvskydmyrw).
+
+### Creating an ML Engine
+
+You can check the available engines with this command:
+
+```sql
+SHOW ML_ENGINES;
+```
+
+If you see the TimeGPT engine on the list, you are ready to follow the tutorials.
+
```

Comment:
We should probably add this, or something similar:

> If you do not see TimeGPT on the list, you will have to create an instance of the engine first with this command:

> `CREATE ML_ENGINE timegpt FROM timegpt USING api_key = '...'`

> Notice that the USING clause is optional, but you will need to pass an API key eventually (either at model creation, engine creation, model usage, or in the mindsdb configuration file). The only exception to this is when using cloud.mindsdb.com.

Reply:
Thank you @paxcema! Fixed in c02d9034dc0a4445a5ec67a466f5c646828293b7.